test/screenshot/spec/mdc-textfield/fixture.js

```diff
@@ -26,5 +26,11 @@ window.mdc.testFixture.fontsLoaded.then(() => {
     mdc.textField.MDCTextField.attachTo(el);
   });
 
+  const firstFocusField = document.querySelector('.mdc-text-field--focused input, .mdc-text-field--focused textarea');
+  if (firstFocusField) {
+    // Programmatically focus to get :focus applied
+    firstFocusField.focus();
```

Comment:
This'll make only the first element in the demo to have focus?

Reply:
Yes. Only one element in the document can have focus at a time, but this is enough to test the textarea issue and I want to see how the screenshot tests come out to see if it can be useful for other text field pages as well.